omod/src/main/webapp/resources/htmlforms/HMIS-ACP-007-MissedAppointmentRegister.xml

```diff
@@ -2,335 +2,281 @@
+					$thisRow = jq(this).parents('div.follow-up:first');
 
-			$thisRow.hide('slow', function() {
-				showAddButtonForLastVisibleRow($thisRow.parents('div.follow-up:first'));
-				fieldHelper.clearAllFields($thisRow);
-			});
-		}
-		});
-		<ifMode mode="ENTER">
-			$otherRows.hide(); //Hide all rows apart from first row if adding new form
-			$firstRow.find('.remove').hide(); //Hide the remove button from the first row
+					$thisRow.hide('slow', function() {
+						showAddButtonForLastVisibleRow($thisRow.parents('div.follow-up:first'));
+						fieldHelper.clearAllFields($thisRow);
+					});
+				}
+				});
+				<ifMode mode="ENTER">
+					$otherRows.hide(); //Hide all rows apart from first row if adding new form
+					$firstRow.find('.remove').hide(); //Hide the remove button from the first row
 
-			jq('.follow-up').each(function(){
-				jq(this).find('[id^="165399"]').find("input[type$='text']").attr("disabled",true);
-			});
+					jq('.follow-up').each(function(){
+						jq(this).find('[id^="165399"]').find("input[type$='text']").attr("disabled",true);
+					});
 
-		</ifMode>
+				</ifMode>
 
-		<!-- Disable the text box for Other(Specify) option if this option is not checked -->
-		<!-- Other Service Point -->
-		jq('#164993').click(function () {
-			if (getValue('164993.value') == 90002) {
-			enableContainer('#' + 165398);
-			} else {
-			disableContainer('#165398');
-			}
-		});
-		<!-- Other Medications due for Refill -->
-		jq('#160288').click(function () {
-			if (getValue('160288.value') == 90002) {
-			enableContainer('#' + 164975);
-			} else {
-			disableContainer('#164975');
-			}
-		});
-		<!-- Disble the options under MCH/MBCP if this option is not checked -->
-		jq('#165348').click(function () {
-			if (getValue('165348.value') == 165348) {
-			enableContainer('#' + 90041);
-			} else {
-			disableContainer('#90041');
-			}
-		});
-		});
+				<!-- Disable the text box for Other(Specify) option if this option is not checked -->
+				<!-- Other Service Point -->
+				jq('#164993').click(function () {
+					if (getValue('164993.value') == 90002) {
+					enableContainer('#' + 165398);
+					} else {
+					disableContainer('#165398');
+					}
+				});
+				<!-- Other Medications due for Refill -->
+				jq('#160288').click(function () {
+					if (getValue('160288.value') == 90002) {
+					enableContainer('#' + 164975);
+					} else {
+					disableContainer('#164975');
+					}
+				});
+				<!-- Disble the options under MCH/MBCP if this option is not checked -->
+				jq('#165348').click(function () {
+					if (getValue('165348.value') == 165348) {
+					enableContainer('#' + 90041);
+					} else {
+					disableContainer('#90041');
+					}
+				});
+			});
 		}
 	</script>
 	<div>
 		<div class="row">
 			<div class="col-md-6">
-				<h3>
-					Missed Appointment Tracking Register
-				</h3>
+				<h1>Missed Appointment Tracking Register</h1>
```

Comment:
@semujju Guidelines state use of H4 here

Reply:
For Forms/Registers` Titles we are using H1 Across, H4 is for card titles.

Reply:
For Forms/Registers` Titles we are using H1 Across, H4 is for card titles.